packages/frontier-plugins/plugin-vue/src/commands/vue/plugin/vuetify.ts

```diff
@@ -5,11 +5,14 @@ const exec = util.promisify(shell.exec);
 import { Command, flags } from '@oclif/command';
 import path from 'path';
 import chalk from 'chalk';
-import { Files } from '../../../modules';
-import { copyFiles, parseDynamicObjects, parseModuleConfig } from '../../../lib/files';
-import { checkProjectValidity, isJsonString } from '../../../lib/utilities';
-import { CLI_COMMANDS, CLI_STATE, DYNAMIC_OBJECTS } from '../../../lib/constants';
-import { injectImportsIntoMain, injectModulesIntoMain } from '../../../lib/plugins';
+import { Files } from 'modules';
+import { isJsonString } from '@rdfrontier/stdlib';
+import { copyFiles, parseDynamicObjects, parseModuleConfig } from '../../../utils/files';
+import { checkProjectValidity } from '../../../utils/utilities';
+import { CLI_COMMANDS, CLI_STATE, DYNAMIC_OBJECTS } from '../../../utils/constants';
+import { injectImportsIntoMain, injectModulesIntoMain } from '../../../utils/plugins';
+import catchFunction from '../../../functions/catch';
```

Comment:
why is this not @rdfrontier/plugin-shared/functions/catch

Reply:
I had it inside the vue folder because it uses CLI_STATE from the vue constants when logging an error message. I can fix it and shift stuff around,